src/verify/test_account_block.sql

```diff
@@ -49,6 +51,24 @@ BEGIN
   contactBB := maevsi_test.contact_select_by_account_id(accountB);
   contactCC := maevsi_test.contact_select_by_account_id(accountC);
 
+  -- A blocks B
+  PERFORM maevsi_test.account_block_create(accountA, accountB);
+
+  BEGIN
+     -- A wants to add B as a contact, should fail
+     contactAB := maevsi_test.contact_create(accountA, 'b@example.com');
+     RAISE EXCEPTION 'should not get here';
```

Comment:
We can merge such comments into output like exceptions to help the future developer who'd be looking to figure out why this test may be failing:

```suggestion
     contactAB := maevsi_test.contact_create(accountA, 'b@example.com');
     RAISE EXCEPTION 'Unexpected success: User should not be able to add a blocked user as a contact';
```

Reply:
I've pushed this change for all three cases, you can resolve once read.